contributors/design-proposals/scheduling/scheduling-policy.md

```diff
@@ -0,0 +1,781 @@
+-  **Default** scheduling components. These list default values to set unless specified in pods definition.
+
+SchedulingPolicy resources are supposed to apply in a deny-all-except approach.
+
+An AdmissionController must be added to the mutating phase to
+-  add default values if unspecified for NodeSelectors, Scheduler-Name, Anti-Affinity rules, Priority class, and Tolerations,
+-  reject pod scheduling if the serviceaccount running the pod is not allowed to specify requested NodeSelectors, Scheduler-Name, Anti-Affinity rules, Priority class, and Tolerations.
+
+All usable scheduling policies (allowed by RBAC) are merged before evaluating if scheduling constraints defined in pods are allowed.
+
+## SchedulingPolicy
+
+Proposed API group: `extensions/v1alpha1`
+
+SchedulingPolicy is a cluster-scoped resource (not namespaced).
+
+### SchedulingPolicy content
+
+SchedulingPolicy spec is composed of optional fields that allow scheduling rules. If a field is absent from a SchedulingPolicy, this schedpol won't allow any item from the missing field.
+
+```yaml
+apiVersion: extensions/valpha1
+kind: SchedulingPolicy
+metadata:
+  name: my-schedpol
+spec:
+  required:
+    schedulerNames:      # Describes schedulers names that are required
+    priorityClasseNames: # Describes priority class names that are required
+    nodeSelectors:       # Describes node selectors that must be used
+    affinities:          # Describes affinities that must be used
+  allowed:
+    schedulerNames:      # Describes schedulers names that are allowed
+    priorityClasseNames: # Describes priority class names that are allowed
+    nodeSelectors:       # Describes node selectors that can be used
+    tolerations:         # Describes tolerations that can be used
+    affinities:          # Describes affinities that can be used
+  default:
+    schedulerName:       # Describes default scheduler name
+    priorityClasseName:  # Describes default priority class name
+    nodeSelector:        # Describes default node selector
+    tolerations:         # Describes default tolerations
+    affinity:            # Describes default affinity
+```
+
+### required
+Elements here are required, pods won't schedule if they aren't present. Also note that if something is required it is also allowed.
+
+### allowed
+Elements here are allowed, the policy allows the presence of these elements. From Pod's perspective, a pod can use one or N of the allowed items.
+
+### default
+If pods do not specify one, the elements here will be added
+
+
+### Scheduler name
+
+If `schedulerNames` is absent from `allowed`, `default` or `required`, no scheduler is allowed by this specific policy.
+
+#### required
+
+Require that pods use either the green-scheduler (which is used by specifying `spec.schedulerName` in pod definition) or the `my-scheduler` scheduler (by specifying `spec.schedulerName: "my-scheduler"`):
+```yaml
+Kind: SchedulingPolicy
```

Comment:
nit: First letters of fields are lowercase. Please apply here and all other fields and examples below.